App/components/Switch/switch.test.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import { themeRender } from '../test-utils'
+import Switch from './switch'
+import { Text } from '../Typography'
+
+describe('Switch component', () => {
+  const mockFn = jest.fn()
+  it('Component with only necessary props matches snapshot', () => {
+    const tree = themeRender(<Switch />).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+  it('Component with all props matches snapshot', () => {
+    const tree = themeRender(
+      <Switch
+        switchType='heart'
+        value
+        onChange={mockFn}
+        onClick={mockFn}
+        disabled
+      >
+        <Text>abc</Text>
+      </Switch>
+    ).toJSON()
+    expect(tree).toMatchSnapshot()
+  })
+  // it('SwitchType prop assigns correctly', () => {
+  //   const elem = themeRender(<Switch switchType='heart' />).toJSON()
+  //   fireEvent.press(elem)
+  //   expect(mockFn).toHaveBeenCalled()
+  // })
```

Comment:
junk

Reply:
I have a problem with getting test instance of the component, so this code is a draft and will be modified when i know how to do it correctly